parm/iodaconv/bufr_ncep_prepbufr_aircraft.yaml

```diff
@@ -0,0 +1,343 @@
+# (C) Copyright 2022 NOAA/NWS/NCEP/EMC
```

Comment:
@spanNOAA Thanks for get all these ready. 
I think now we will use the RDASapp to save these yaml files.

You can check the GDASapp we have on Hera (ufs-ar/arfs/sorc/gdas.cd) and make a similar arrangement.